cmd/adsysd/integration_tests/adsys_test.go

```diff
@@ -79,7 +79,7 @@ func TestMain(m *testing.M) {
 func TestStartAndStopDaemon(t *testing.T) {
 	systemAnswer(t, "polkit_yes")
 
-	conf := createConf(t, "")
+	conf := createConf(t, "", "sssd")
```

Comment:
This is set on every tests, and the number of induced changes triggers me that it’s maybe a little bit too much.

I think this is a good use case for optional function parameter? (Maybe doing the same for adsysDir too while we are at it). (Default being sssd ofc)